### PR TITLE
hyper 0.13 -> 0.14, tonic 0.2 -> 1.0

### PR DESCRIPTION
**Pull request by amilkov3**

### CODE REVIEW

**Review by akshayknarayan — CHANGES_REQUESTED** [review]
Thanks! Looks like the tests still need to be fixed?

**Comment by amilkov3** [comment]
Totally slipped my mind. Fixed

**Comment by akshayknarayan** [comment]
Great, looks good. Resolves #2

File: Cargo.toml [hunk 1]
```diff
@@ -10,8 +10,8 @@ license = "ISC"
 
 [dependencies]
 anyhow = "1.0"
-futures-util = "0.3.1"
+futures-util = "0.3"
 hex = "0.4"
-hyper = { version = "0.13", features = ["stream"] }
-tokio = { version = "0.2", features = ["uds"] }
-pin-project = "0.4"
+hyper = { version = "0.14", features = ["full"] }
```
Comment (fanatid) [inline Cargo.toml:15]: Do we really need `features = ["full"]`?

Comment (fanatid) [reply Cargo.toml:15]: I created pull request with removing not required features: #4

**Pull request merged**